packages/ui/src/components/organisms/WorkflowsTable/WorkflowsTable.tsx

```diff
@@ -0,0 +1,122 @@
+import { Table } from '@components/atoms/Table/Table';
+import { TableBody } from '@components/atoms/Table/TableBody';
+import { TableCell } from '@components/atoms/Table/TableCell';
+import { TableHead } from '@components/atoms/Table/TableHead';
+import { TableHeader } from '@components/atoms/Table/TableHeader';
+import { TableRow } from '@components/atoms/Table/TableRow';
+import { useMemo } from 'react';
+import { useReactTable, flexRender, getCoreRowModel } from '@tanstack/react-table';
+import { defaultColumns } from './columns';
+import { InputColumn, WorkflowsTableSorting, WorkflowTableColumnDef } from './types';
+import keyBy from 'lodash/keyBy';
+import { mergeColumns } from './utils/merge-columns';
+import { WorkflowTableItem } from './types';
+import { TableContainer } from './components/TableContainer';
+import { ScrollContainer } from './components/ScrollContainer';
+
+interface Props {
+  items: WorkflowTableItem[];
+  sorting?: WorkflowsTableSorting;
+  isFetching?: boolean;
+  columns?: InputColumn[];
+  onSort: (key: string, direction: 'asc' | 'desc') => void;
+}
+
+export function WorkflowsTable({ items, isFetching, sorting, columns, onSort }: Props) {
```

Comment:
Is there room for a generic table component and `WorkflowsTable` could be built on top of it?